extension/package.nls.json

```diff
@@ -75,6 +75,8 @@
   "command.views.experimentsTree.selectExperiments": "Select Experiments to Display in Plots",
   "command.views.plotsPathsTree.selectPlots": "Select Plots to Display",
   "command.views.plotsPathsTree.refreshPlots": "Refresh Plots for Selected Experiments",
+  "config.focusedProjects.description": "A subset of paths to the workspace's available DVC projects. Using this option will override project auto-discovery. Only absolute paths inside of the workspace will be included. All other DVC projects in the workspace will be ignored.",
```

Comment:
does it have to be an absolute path?

Reply:
Yes, all of the paths that we accept currently have to be absolute.

Reply:
any specific reason for this? can we resolve from the project / workspace root?

Reply:
One reason is that there can be collisions when a user is using a mutli-root workspace. I.e a relative path can exist in both.